src/site/README.md

```diff
@@ -281,14 +281,15 @@ _Note: you may have noticed that the project also has a `yarn watch:content` tas
 
 This approach is recommended if you are making heavy changes to a template or are writing a template for a page-type without any published instances. The preview server will load data fresh from the CMS on each page request and processes it through the template at that time, meaning you don't need to wait for a content build to run to see your changes.
 
-1. Run `yarn build:webpack --env.entry=static-pages` to generate the JS/CSS files required by templates. If you make changes to the `static-pages` application code, you will need to run this command again.
+1. Run `yarn build:content` and `yarn build:webpack --env.entry=static-pages` to generate the JS/CSS files required by templates. If you make changes to the `static-pages` application code, you will need to run this command again.
```

Comment:
Should we specify which command needs to be run again since we're now mentioning two commands? Maybe something like this.
```suggestion
1. Run `yarn build:content` and `yarn build:webpack --env.entry=static-pages` to generate the JS/CSS files required by templates. If you make changes to the `static-pages` application code, you will need to run the second command again.
```